tests/resolvers/UserTag/childTags.spec.ts

```diff
@@ -0,0 +1,72 @@
+import "dotenv/config";
+import { childTags as childTagsResolver } from "../../../src/resolvers/UserTag/childTags";
+import type {
+  UserTagsConnectionResult,
+  UserTagChildTagsArgs,
+} from "../../../src/types/generatedGraphQLTypes";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { beforeAll, afterAll, describe, it, expect } from "vitest";
+import type { TestUserTagType } from "../../helpers/tags";
+import { createTwoLevelTagsWithOrg } from "../../helpers/tags";
+import { MAXIMUM_FETCH_LIMIT } from "../../../src/constants";
+
+let MONGOOSE_INSTANCE: typeof mongoose;
+let testChildTag1: TestUserTagType,
+  testChildTag2: TestUserTagType,
+  testParentTag: TestUserTagType;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  [, , [testParentTag, testChildTag1, testChildTag2]] =
+    await createTwoLevelTagsWithOrg();
+});
+
+afterAll(async () => {
+  await disconnect(MONGOOSE_INSTANCE);
+});
+
+describe("resolvers -> Tag -> childTags", () => {
+  it(`returns error object when the arguments are incorrect`, async () => {
+    const parent = testParentTag!;
+
+    const args: UserTagChildTagsArgs = {
+      input: {
+        limit: MAXIMUM_FETCH_LIMIT + 1,
+        direction: "FORWARD",
+      },
+    };
+
+    const payload = await childTagsResolver?.(parent, args, {});
+
+    expect(payload!.errors.length).toEqual(1);
```

Comment:
you have to be explicit and precise about the error that's going to be thrown, it cannot be disregarded as just an error, you at least need to check the`__typename`.